sit/launch_review_job.py

```diff
@@ -213,7 +236,7 @@ def get_cluster(self):
         try:
             return self.cf_helper.get_resource_name(self.STACK_NAME, self.LOGICAL_CLUSTER_NAME)
         except Exception as e:
-            self.error('Failed to retrieve the cluster', e)
+            self.error('Failed to retrieve the cluster with error: {0}'.format(e))
```

Comment:
There is no need to do this. original method of passing the exception works. Please look at:
https://github.com/dandb/salt-integration-testing/pull/20/files#diff-bde3ecd9cba050aae35eef1675145a0dR328